wiki/CreatingMissions.md

```diff
@@ -327,24 +327,24 @@ This specifies that the mission does not show up in the player's list of mission
 (priority | minor | non-blocking)
 ```
 
-If a mission is marked with `priority`, only other "priority" missions can be offered alongside it.
+If a mission is marked with `priority`, only other "priority" missions can be offered alongside it, except, beginning in **v. 0.10.13**, missions marked `non-blocking`.
```

Comment:
```suggestion
If a mission is marked with `priority`, only other "priority" missions and `non-blocking` missions can be offered alongside it.
```

Reply:
This hasn't always been the behaviour; we still need to include the version number.